presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -5988,6 +5989,12 @@ public void TestTryExpression()
         assertQuery("SELECT 123 + TRY(ABS(-9223372036854775807 - 1))", "SELECT NULL");
         assertQuery("SELECT JSON_FORMAT(TRY(JSON '[]')) || '123'", "SELECT '[]123'");
         assertQuery("SELECT JSON_FORMAT(TRY(JSON 'INVALID')) || '123'", "SELECT NULL");
+
+        // Tests from TestExpressionInterpreter.java
```

Comment:
I think we can discard these tests :). Since originally they are used to test `ExpressionInterpreter`, and those special treatment are now removed :)

Reply:
Update: Discussed  with @haozhun , after your change, the only place that `ExpressionInterpreter` will be used before the Desugaring happens is in `StatemnetAnalyzer.visitSampledRelation`, which is weird to use a try expression.

Thus we decided it is fine that ExpressionInterpreter no longer handles the TRY expression before desugaring. As a result, these `assertOptimizedEquals` tests are no longer necessary.

However, these tests can still be extracted to test the TRY expression alone, as you did for some of them. I think you miss  `"42 + TRY(ABS(-9223372036854775807 - 1))` ? :P. Just double check all valuable test cases are retained :)